en/developer_guide/entity_list_ajax_pagination.rst

```diff
@@ -13,8 +13,39 @@ the plugin makes an ajax request to the api with a number of parameters.
 Usage
 -----
 
-The different variables used to filter (pagination, search, ordering) the data
-are described in this `documentation`_.
+The different variables used to filter (pagination, search, ordering) send by the plugin are:
+
++-----------------------+--------------------------------------------------+
+| **Parameter name**    | **Description**                                  |
++-----------------------+--------------------------------------------------+
+| order[name]           | Column's name to which ordering                  |
+|                       | should be applied                                |
++-----------------------+--------------------------------------------------+
+| order[dir]            | Ordering direction (asc or desc)                 |
++-----------------------+--------------------------------------------------+
+| search[columns][name] | Search value to apply to a specific column       |
+|                       | Name is the ``name`` of column                   |
++-----------------------+--------------------------------------------------+
+| search[global]        | Global search value                              |
++-----------------------+--------------------------------------------------+
+| length                | Number of records that the table                 |
+|                       | can display in the current draw                  |
++-----------------------+--------------------------------------------------+
+| start                 | Paging first record indicator                    |
++-----------------------+--------------------------------------------------+
+
+If there isn't value in input of search (global and columns) the parameter ``search`` is not sent.
```

Comment:
What about : 

The parameter ``search`` is not present in the request if there is no value in any input field